opa-manager-operator/README.md

```diff
@@ -0,0 +1,79 @@
+# Kubernetes Open Policy Agent Operator
+
+
+## Description
+
+This repo contains the charm for the Audit agent operator
+
+## Usage
+
+### bootstrap (if new to juju)
+
+
+```
+$ sudo snap install juju --classic
+
+$ k8s_name=yournaminghere
+
+$ juju add-k8s ${k8s_name}-cloud
+
+$ juju bootstrap ${k8s_name}-cloud ${k8s_name}
+
+```
+
+To deploy run:
+
+### Add Model
+
+```
+$ NAMESPACE=gatekeeper
+$ juju add-model ${NAMESPACE}
+
+```
+
+
+###  Deploy
+
+```
+$ juju deploy gatekeeper-audit --channel=beta
+$ juju deploy gatekeeper-manager --channel=beta
+
+```
+
+
+### Post deployment steps
+
+```
+$ kubectl apply -f docs/gatekeeper-rb.yaml
+$ CA_CERT=$(kubectl get secrets -n gatekeeper gatekeeper-webhook-server-cert -o jsonpath="{.data.ca\.crt}")
+
+$ CA_CERT=$(kubectl get secrets -n ${NAMESPACE} gatekeeper-webhook-server-cert -o jsonpath="{.data.ca\.crt}")
+$ for i in {0..1}; do kubectl patch validatingWebhookConfigurations ${NAMESPACE}-gatekeeper-validating-webhook-configuration --type='json' -p='[{"op": "replace", "path": "/webhooks/'"$i"'/clientConfig/caBundle", "value":'"${CA_CERT}"'}]'; done
+$ kubectl apply -f docs/gatekeeper-rb.yaml
```

Comment:
CA_CERT defined twice? gatekeeper-rb.yaml applied twice?

Reply:
This still applies docs/gatekeeper-rb.yaml twice, please update.

Reply:
TODO: Update instructions here and on audit operator

Reply:
updated